goreleaser/configure.go

```diff
@@ -0,0 +1,223 @@
+//go:build releaser
+
+package main
+
+// This file is a script which generates the .goreleaser.yaml file for all
+// supported OpenTelemetry Collector distributions.
+//
+// Run it with `make generate-goreleaser`.
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/goreleaser/goreleaser/pkg/config"
+	"github.com/goreleaser/nfpm/v2/files"
+	yaml "gopkg.in/yaml.v2"
+)
+
+var (
+	ImagePrefixes = []string{"otel"}
```

Comment:
I made this a slice in order to pave the way for #49. We’ll be able to publish the images to additional registries by just adding them here (and adding their authentication to CI).